#include <string>
using namespace  std;
string log_python_exception();
void LogPythonError();
